addons/website_slides/static/src/interactions/slide_like.js

```diff
@@ -0,0 +1,75 @@
+import { Interaction } from "@web/public/interaction";
+import { registry } from "@web/core/registry";
+
+import { sprintf } from '@web/core/utils/strings';
+import { _t } from "@web/core/l10n/translation";
+import { rpc } from "@web/core/network/rpc";
+
+export class SlideLike extends Interaction {
+    static selector = "#wrapwrap";
```

Comment:
I guess the selector should be `.o_wslides_js_slide_like` to reproduce what was going on in `publicWidget.registry.websiteSlidesSlideLike` ?